docker/experimental/ansible/roles/fetch_srcs/tasks/main.yaml

```diff
@@ -34,8 +34,10 @@
     # localhost.
     remote_src: true
     strip: 1
+    ignore_whitespace: true
     basedir: "{{ (src_root, 'pytorch/xla/third_party/tensorflow') | path_join }}"
```

Comment:
This is now gone, and bazel does the patching.

Reply:
But it's not gone for v1.12 and v.1.13 releases? The problem with this approach is that build & deployment configuration is not bound with sources version (and for now I don't think we can improve on that). So this build process has to work for current and future versions of the code as well as older ones. 

Do you think that having this step while bazel is doing the patching is problematic? I added `ignore_errors: true` so if the step fails it's fine, the playbook still continues.